lib/mixlib/install/generator/powershell/scripts/helpers.ps1.erb

```diff
@@ -112,22 +112,13 @@ function Test-ProjectPackage {
   param ($Path, $Algorithm = 'SHA256', $Hash)
 
   if (-not (get-command get-filehash -ErrorAction 'SilentlyContinue')) {
```

Comment:
I removed this if statement and local Get-FileHash function because we are using this inside the if block only. Another Get-FileHash is powershell cmdlet.